data/quests.ts

```diff
@@ -1,6 +1,109 @@
 import { ListType } from "./types";
 
 export const Quests: Array<ListType> = [
+  /* TEMPLATE     
+  {
+    // Can be left blank to be filled later
+    id: "",
+
+    // Name of the NPC/ending
+    name: "",
+
+    requirements: [
+      {
+        // Can be left blank to be filled later
+        id: "",
+
+        // Short sentence in brief describing necessary actions with few details
+        // Acceptable details:
+        // - Nearby Site of Grace, or Map Section
+        // - Necessary bosses and items
+        // - Dialog options
+        description: ""
+      },
+    ]
+  },
+  */
+  {
+    id: "",
+    name: "Kenneth Haight (Incomplete)",
+    requirements: [
+      {
+        id: "",
+        description: "Talk to Kenneth in West Limgrave near Third Church of Marika site of grace.",
+      },
+      {
+        id: "",
+        description: "Clear out enemies in Fort Haight in West Limgrave.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at his first location. Receive a reward.",
+      },
+      {
+        id: "",
+        description: "Talk to Kenneth at Fort Haight.",
+      },
+      {
+        id: "",
+        description: "To Be Determined ...",
+      },
+    ],
+  },
+  {
+    id: "",
+    name: "Nepheli Loux (Incomplete)",
```

Comment:
I've added what information is currently known about this questline. So I'd leave this without the incomplete line. If anything else is found about this, then it can be added later.
```suggestion
    id: "",
    name: "Nepheli Loux",
```